api-reference/v2/tables/get-table-rows.mdx

```diff
@@ -2,3 +2,7 @@
 title: Get Rows
 openapi: get /tables/{tableID}/rows
 ---
+
+You can use the `limit` query parameter to specify the maximum number of rows to return for each request.
```

Comment:
We should specify what the default limit value is. Also, what is the max value the limit can contain. I assume if they pass 1,000,000 we're not going to honor it?